test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable protractor/no-describe-selectors */
+import { browser, by } from 'protractor';
+import { config } from '../protractor/local.config';
+
+
+describe('Buy a t-shirt', () => {
+
+  beforeEach(async() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
+
+    const username = 'standard_user';
+    const passwd = 'secret_sauce';
+
+    const firstname = 'Dennys';
+    const lastname = 'Mosquera';
+    const zipcode = '720016';
+
+    await browser.wait(browser.get(config.baseBuyUrl), 10000, "Not possible to access 1.");
+
+    const user_ele = browser.findElement(by.id('user-name'));
+    const pass_ele = browser.findElement(by.id('password'));
+    const loginbtn_ele = browser.findElement(by.id('login-button'));
```

Comment:
Don't combine camel case with snake case
and also put well camel case `loginBtn`  for example